.github/workflows/main.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Push to ECR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+      
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      
+      - name: Login to Amazon ECR
+        uses: aws-actions/amazon-ecr-login@v1
+      
+      - name: Build and push to Amazon ECR
+        env:
+          DOCKER_IMAGE: ${{ secrets.DOCKER_IMAGE }}
+        run: |
+          COMMIT_HASH=$(git rev-parse --short HEAD)
+          docker build --platform linux/amd64 -t $DOCKER_IMAGE:$COMMIT_HASH -f Dockerfile .
```

Comment:
In the future when we start introducing API Keys and FTP credentials, are we going to store those at github variables? If so, will we need to manually update build args here?

Reply:
Good point! After some back and forth, instead of using GitHub variables, I was thinking we'll store sensitive information like API keys and FTP credentials in AWS Secrets Manager (with respective scripts for this to come in future PRs). Let me know if you suggest otherwise or see any issues with this!

Reply:
Yeah that sounds like a perfect plan! I was going to say secrets manager or ejson might be the move but I realize decrypting, editing, re-encrypting the ejson (which requires having the keys locally) is going to get annoying over time as opposed to just retrieving it from AWS

Reply:
I would be hesitant to use ejson on our forked repo